src/App.tsx

```diff
@@ -1,26 +1,93 @@
-import React from "react";
-import logo from "./logo.svg";
+import { useState } from "react";
 import "./App.css";
+import { InputTask } from "./components/listItem";
+import { MyButton } from "./components/myButton";
+import { MySearching } from "./components/MySearching";
 
 function App() {
+  const [tasks, setTasks] = useState<{ isDone: boolean; title: string }[]>([]);
+  const [inputValue, setInputValue] = useState<string>("");
+  const [searchValue, setSearchValue] = useState<string>("");
+
+  const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setInputValue(e.target.value);
+  };
+
+  const handleCreateTask = () => {
+    if (inputValue.trim() === "") return;
```

Comment:
next task for you, add `toast` library(just node module from npm) and throw notification instead of empty return